CONTRIBUTING.md

```diff
@@ -64,11 +116,73 @@ To start the backend with a debugger run the [`main.go`-file](./main.go) located
 
 Make sure to use the following configurations:
 
-TODO document workflow
+<!-- TODO: document workflow -->
 
 ### Console
```

Comment:
@eliobischof probably we should also make a quick note about clean-up / teardown. 
We start for example containers in --detach mode. Would be nice to provide also the commands to clean-up, just QoL.

Reply:
done